lib/tasks/job_fetch.rake

```diff
@@ -1,15 +1,11 @@
 namespace :job_fetch do
   desc "Run the stack overflow fetcher"
 
-  task stackoverflow: :environment do
+  task all_jobs: :environment do
```

Comment:
This is probably the easiest way to make sure we have one rake task for Heroku Scheduler.